crates/aptos-logger/Cargo.toml

```diff
@@ -22,11 +25,13 @@ serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.81"
 strum = "0.24.1"
 strum_macros = "0.24.2"
+tokio = { version = "1.18.2", features = ["full"] }
 tracing = "0.1.34"
 tracing-subscriber = "0.3.11"
 
-aptos-infallible = { path = "../aptos-infallible" }
```

Comment:
FYI: we do this on purpose. The convention is that dependencies are grouped by third-party dependencies and internal dependencies (just makes the code easier to reason about) 😄

Reply:
It movbed because of 'error: Dependencies for aptos-logger are not sorted'

Reply:
This is fine, as long as there's a gap between the two lists. Our CI/CD invokes the `--grouped` flag. See the CI/CD script:

https://github.com/aptos-labs/aptos-core/blob/0ff8166e6f289e2cddbe74e3a4139fb8e50ff019/.github/workflows/lint-test.yaml#L128